frontend/pages/beta/index.tsx

```diff
@@ -0,0 +1,164 @@
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import Typography from '@mui/material/Typography'
+import Link from 'next/link'
+import React, { useState } from 'react'
+import TagSelector from 'src/common/TagSelector'
+
+import { ListModelType, useListModels } from '../../data/model'
+import EmptyBlob from '../../src/common/EmptyBlob'
+import MultipleErrorWrapper from '../../src/errors/MultipleErrorWrapper'
+import Wrapper from '../../src/Wrapper'
+import { Model, Version } from '../../types/types'
+import useDebounce from '../../utils/hooks/useDebounce'
+
+export default function ExploreModels() {
+  const [group, setGroup] = useState<ListModelType>('all')
+  const [filter, setFilter] = useState('')
+  const debouncedFilter = useDebounce(filter, 250)
+
+  const { models, isModelsError, mutateModels } = useListModels(group, debouncedFilter)
+
+  const theme = useTheme()
+
+  const error = MultipleErrorWrapper(`Unable to load marketplace page`, {
+    isModelsError,
+  })
+  if (error) return error
+
+  const handleFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFilter(e.target.value)
+  }
+
+  const onFilterSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+  }
+
+  const updateSelectedTags = (selectedTags: string[]) => {
+    console.log(selectedTags)
+  }
+
+  const updateSelectedType = (selectedType: string) => {
+    if (selectedType) {
+      switch (selectedType) {
+        case 'My models':
+          setGroup('user')
+          break
+        case 'Favourites':
+          setGroup('favourites')
+          break
+        default:
+          setGroup('all')
+      }
+      mutateModels()
+    }
+  }
+
+  return (
+    <Wrapper title='Explore Models' page='marketplace'>
+      <Paper
+        component='form'
+        onSubmit={onFilterSubmit}
+        sx={{
+          p: '2px 4px',
+          display: 'flex',
+          alignItems: 'center',
+          width: '70%',
+          maxWidth: '400px',
+          marginBottom: 3,
+        }}
+      >
+        <InputBase sx={{ ml: 1, flex: 1 }} placeholder='Filter Models' value={filter} onChange={handleFilterChange} />
+        <IconButton color='primary' type='submit' sx={{ p: '10px' }} aria-label='filter'>
+          <SearchIcon />
+        </IconButton>
+      </Paper>
+      <Grid container>
+        <Grid item sm={8} xs={12}>
+          <Paper sx={{ py: 2, px: 4 }}>
+            <Box sx={{ mb: 2, borderBottom: 1, borderColor: 'divider' }} data-test='indexPageTabs'>
+              <Tabs value={'bailo'}>
+                <Tab label={`Models ${models ? `(${models.length})` : ''}`} value='bailo' />
+              </Tabs>
+            </Box>
+            <Box data-test='modelListBox'>
+              {models &&
+                models.map((model: Model, index: number) => {
+                  const latestVersion = model.latestVersion as Version
+                  return (
+                    <Box key={model.uuid}>
```

Comment:
This could be a `<Fragment key={model.uuid}>` as you're not using any mui utilities